indra/newview/llinventorymodel.cpp

```diff
@@ -1290,7 +1290,6 @@ void LLInventoryModel::collectDescendentsIf(const LLUUID& id,
         }
     }
 
-    LLViewerInventoryItem* item = NULL;
```

Comment:
Thanks! I didn't see this in the build log for some reason.

Reply:
I didn't break the Windows build though...

Reply:
Right, it looks like GitHub is trying to build the non-opensource version on your external repo.
I'll test it on my dev setup later today.